ramin/js/basic-syntax-35/ramin.js

```diff
@@ -0,0 +1,164 @@
+// add number a to b
+function add(a, b) {
+    return a+b;
+}
+
+/// ----------------------------------- ////
+
+// flip a coin and log the result (head or tail)
+function flipACoin() {
+    //Math.random() generate a 0<=floating_point_number<1 
```

Comment:
Do not leave these kinds of comments in your code. Anyone familiar with JS knows what `Math.random` does.